dfget/core/core.go

```diff
@@ -262,6 +262,10 @@ func checkConnectSupernode(nodes []string) (localIP string) {
 
 func calculateTimeout(fileLength int64, defaultTimeoutSecond int, minRate int) time.Duration {
 	timeout := 5 * 60
+	// avoid trigger panic when minRate equals zero
+	if minRate == 0 {
```

Comment:
Is it better for `if minRate <= 0 ` ?

Reply:
Is it possible for a client to provide a negative value for minRate?

Reply:
Here is the definition `flagSet.StringVar(&minRate, "minrate", "",		flagSet.StringVar(&minRate, "minrate", "", "minimal network bandwidth rate for downloading a file, in format of 20M/m/K/k")`

And I think `<= 0` could get more assurance.

Reply:
Done.